config/sequelize.js

```diff
@@ -2,6 +2,7 @@ exports.default = {
     sequelize: function(api){
         return {
             "autoMigrate" : true,
+            "loadFixtures": true,
```

Comment:
Yeah, that's OK.  I think it's the right thing to do.  You should only want fixtures within a `test` environment.